frappe/patches/v12_0/move_email_and_phone_to_child_table.py

```diff
@@ -0,0 +1,35 @@
+import frappe
+
+def execute():
+	contact_details = frappe.get_list("Contact", fields=["name", "email_id", "phone", "mobile_no", "modified_by", "creation", "modified"])
+	frappe.reload_doc("contacts", "doctype", "contact_email")
+	frappe.reload_doc("contacts", "doctype", "contact_phone")
+	frappe.reload_doc("contacts", "doctype", "contact")
+
+	name_counter = 100000000
+	for contact_detail in contact_details:
+		contact_name = contact_detail.name.replace('"', '\\"').replace("'", "\\'")
+
+		if contact_detail.email_id:
+			frappe.db.sql("""
+				INSERT INTO `tabContact Email`
+					(`idx`, `name`, `email_id`, `parentfield`, `parenttype`, `parent`, `is_primary`, `creation`, `modified`, `modified_by`)
+				VALUES (1, "{0}", "{1}", "email_ids", "Contact", "{2}", 1, "{3}", "{4}", "{5}")
```

Comment:
```suggestion
				VALUES (1, '{0}', '{1}', 'email_ids', 'Contact', '{2}', 1, '{3}', '{4}', '{5}')
```
Since Postgres considers value inside double quotes as identifier